### ISSUE TITLE
accuracy is 0

### ISSUE THREAD

**Issue by NicoleNicoleXiang** [sirius-ai/MobileFaceNet_TF#39]
hi：
   the problem is accuracy is always 0 when i applying train_net.py,meanwhile,the loss is decreasing and accuracy on lfw is incresing.
   i think that i applyed the one you revised at num_class..look at the error below:

class_number:85742(MS1M-V2)

epoch 0, total_step 1250, total loss is 42.33 , inference loss is 42.01, reg_loss is 0.32, training accuracy is 0.000000, time 323.625 samples/sec
epoch 0, total_step 1300, total loss is 42.36 , inference loss is 42.03, reg_loss is 0.32, training accuracy is 0.000000, time 319.744 samples/sec
epoch 0, total_step 1350, total loss is 42.08 , inference loss is 41.76, reg_loss is 0.32, training accuracy is 0.000000, time 322.450 samples/sec
epoch 0, total_step 1400, total loss is 42.24 , inference loss is 41.92, reg_loss is 0.32, training accuracy is 0.000000, time 319.986 samples/sec
epoch 0, total_step 1450, total loss is 42.51 , inference loss is 42.19, reg_loss is 0.32, training accuracy is 0.000000, time 322.289 samples/sec
epoch 0, total_step 1500, total loss is 42.49 , inference loss is 42.17, reg_loss is 0.32, training accuracy is 0.000000, time 318.947 samples/sec
epoch 0, total_step 1550, total loss is 42.36 , inference loss is 42.04, reg_loss is 0.32, training accuracy is 0.000000, time 314.517 samples/sec
epoch 0, total_step 1600, total loss is 42.45 , inference loss is 42.14, reg_loss is 0.31, training accuracy is 0.000000, time 324.133 samples/sec
epoch 0, total_step 1650, total loss is 42.35 , inference loss is 42.03, reg_loss is 0.31, training accuracy is 0.000000, time 322.223 samples/sec
epoch 0, total_step 1700, total loss is 42.44 , inference loss is 42.12, reg_loss is 0.31, training accuracy is 0.000000, time 323.349 samples/sec

thresholds max: 0.71 <=> min: 0.49
total time 13.695s to evaluate 12000 images of lfw
Accuracy: 0.716+-0.016
Validation rate: 0.03367+-0.01538 @ FAR=0.00100
fpr and tpr: 0.725 0.875
Area Under Curve (AUC): 0.799
Equal Error Rate (EER): 0.275

**Comment by jolinlinlin** [comment]
I have the same problem with you when I run python train_nets.py --class_number=85742,the epoch and training accuracy are always 0,only the reg_loss is decreasing,but accuracy on lfw is incresing.Have you solved this problem?Please tell me details.Thank you very much!

**Comment by CHANGErIBE** [comment]
just keep running

**Comment by jolinlinlin** [comment]
> just keep running

I keep running for 11 epochs and accuracy in validation set rise to 99.5%,but training accuracy is still always 0. Do you know why? Thank you very much!

**Comment by sirius-ai** [comment]
same with [#40](https://github.com/sirius-ai/MobileFaceNet_TF/issues/40)
